src/Symfony/Component/DependencyInjection/Tests/Fixtures/includes/autowiring_classes_80.php

```diff
@@ -125,3 +126,17 @@ public function __construct(
     {
     }
 }
+
+class LazyServiceAttributeAutowiring
+{
+    public function __construct(#[Lazy] A $a)
+    {
+    }
+}
+
+class LazyAutowireServiceAttributesAutowiring
+{
+    public function __construct(#[Lazy]#[Autowire(lazy: true)] A $a)
```

Comment:
```suggestion
    public function __construct(#[Lazy, Autowire(lazy: true)] A $a)
```

Reply:
unresolving as the suggestion has not been applied and I see no reason why

Reply:
Sorry, mistook it for the other one. Fixed!